lib/common/models/collection.rb

```diff
@@ -26,18 +26,66 @@ class Collection
       'match' => 'match'
     }.with_indifferent_access.freeze
 
-    def initialize(klass = Array, data: [], metadata: {}, errors: {})
+    def initialize(klass = Array, data: [], metadata: {}, errors: {}, cache_key: nil)
       data = Array.wrap(data) # If data is passed in as nil, wrap it as an empty array
       @type = klass
       @attributes = data
       @metadata = metadata
       @errors = errors
+      @cache_key = cache_key
       (@data = data) && return if defined?(::WillPaginate::Collection) && data.is_a?(WillPaginate::Collection)
       @data = data.collect do |element|
         element.is_a?(Hash) ? klass.new(element) : element
       end
     end
 
+    def self.redis_namespace
+      @redis_namespace ||= Redis::Namespace.new('Common::Collection', redis: Redis.current)
```

Comment:
Yea, I suppose it wouldn't hurt to have defaults specified in redis.yml that this uses, that way redis.yml still draws attention to the fact that this namespace exists. I'll make that change, the key and TTL will still be able to be overridden though on case by case basis.